rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/FilteredTopWorkflowsNumberOfNodesData.java

```diff
@@ -31,7 +31,7 @@ public class FilteredTopWorkflowsNumberOfNodesData {
 
     private String projectName;
 
-    private int numberOfNodes;
+    private double numberOfNodes;
```

Comment:
I guess double is because you use the AVG function in the db.

I would prefer that you keep this to int and that you apply the ROUND SQL function in the query (the same applies to other similar queries)